CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+bug report and let us know!
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Github issues
+* Communications / Social Media / Blog Posts
+* Release management
+
+## Find an Issue
+
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. [good first issue](https://github.com/coralogix/telemetry-shippers/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) has extra information to
+help you make your first contribution. [help wanted](https://github.com/coralogix/telemetry-shippers/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) are issues
+suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
+
+Sometimes there won’t be any issues with these labels. That’s ok! There is
+likely still something for you to work on. If you want to contribute but you
+don’t know where to start or can't find a suitable issue, you can ⚠️ **explain how people can ask for an issue to work on**.
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it. Something like "I want to work on this" is fine.
+
+## Ask for Help
+
+The best way to reach us with a question when contributing is to ask on the original github issue.
+
+## Pull Request Lifecycle
+
+⚠️ **Explain your pull request process**
+
+[TODO] We need define a PR Lifecycle.
+
+## Development Environment Setup
+
+[TODO] Should we provide local environments?
+
+⚠️ **Explain how to set up a development environment**
+
+## Sign Your Commits
+
+⚠️ **Keep either the DCO or CLA section depending on which you use**
+
+### DCO
+Licensing is important to open source projects. It provides some assurances that
+the software will continue to be available based under the terms that the
+author(s) desired. We require that contributors sign off on commits submitted to
+our project's repositories. The [Developer Certificate of Origin
+(DCO)](https://probot.github.io/apps/dco/) is a way to certify that you wrote and
+have the right to contribute the code you are submitting to the project.
+
+You sign-off by adding the following to your commit messages. Your sign-off must
+match the git user and email associated with the commit.
+
+    This is my commit message
+
+    Signed-off-by: Your Name <your.name@example.com>
+
+Git has a `-s` command line option to do this automatically:
+
+    git commit -s -m 'This is my commit message'
+
+If you forgot to do this and have not yet pushed your changes to the remote
+repository, you can amend your commit with the sign-off by running 
+
+    git commit --amend -s 
+
+### CLA
+We require that contributors have signed our Contributor License Agreement (CLA). 
+
+⚠️ **Explain how to sign the CLA**
+
+## Pull Request Checklist
```

Comment:
Same as PR Lifecycle.

Reply:
See my other comment, we could keep this simple for now.